### ISSUE TITLE
compact bone area of femur

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3639]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

    1. decreased femoral compact bone area 
    2. increased femoral compact bone area 
    

>2. Synonyms (e.g., absent spleen)

   1. reduced femoral compact bone area, decreased compact bone area of femur, decreased compact bone cross-sectional area of femur
   2. increased compact bone area of femur, increased compact bone cross-sectional area of femur

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  reduction in the total amount of cross-sectional area of compact bone tissue in the femur
2.  increase in the total amount of cross-sectional area of compact bone tissue in the femur

PMID:22446865

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. decreased compact bone area (MP:0010968)
2. increased compact bone area (MP:0010967)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
Added terms

MP:0021182 | decreased femoral compact bone area
MP:0021183 | increased femoral compact bone area
